PSReadLine.build.ps1

```diff
@@ -138,10 +94,27 @@ Synopsis: Run the unit tests
 #>
 task RunTests BuildMainModule, BuildXUnitTests, { Start-TestRun -Configuration $Configuration -Framework $Framework }
 
+<#
+Synopsis: Check if the help content is in sync.
+#>
+task CheckHelpContent {
+    if ($CheckHelpContent) {
```

Comment:
See https://github.com/nightroman/Invoke-Build/wiki/Script-Tutorial#conditional-tasks on how to specify a conditional task.